From 567865310968110fdbe77a7f8aae11fd511e2beb Mon Sep 17 00:00:00 2001
From: HenryXin <yuepeng.xin@gmail.com>
Date: Wed, 14 Jan 2026 04:24:07 +0000
Subject: [PATCH] a test patch to main.c

---
 init/main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/init/main.c b/init/main.c
index c787e94cc898..0f6f3a715793 100644
--- a/init/main.c
+++ b/init/main.c
@@ -904,6 +904,8 @@ void start_kernel(void)
 	smp_prepare_boot_cpu();	/* arch-specific boot-cpu hooks */
 	boot_cpu_hotplug_init();
 
+	printk(KERN_NOTICE "Patched: Kernel command line: %s\n", saved_command_line);
+	
 	pr_notice("Kernel command line: %s\n", saved_command_line);
 	/* parameters may set static keys */
 	jump_label_init();
-- 
2.44.4

